src/reducers/sidebar.reducer.js

```diff
@@ -34,6 +38,26 @@ export default function sidebar(state = initialState.sidebar, action) {
         ...state,
         inputStoryName: action.payload
       }
+    case POIS_INPUT_UPDATED:
+      return {
+        ...state,
+        inputPois: action.payload
+      }
+    case EDITING_POIS_LOADED:
```

Comment:
since these two actions have the same effect, you can cascade the switch cases:
```javascript
case POIS_INPUT_UPDATED:
case EDITING_POIS_LOADED:
  return {
    ...state,
    inputPois: action.payload
  }
```